### ISSUE TITLE
Broken symlinks in repo

### ISSUE THREAD

**Issue by chdiza** [Homebrew/homebrew#43600]
I wonder if anyone can reproduce this, but in two separate clones on my machine, the following files are symlinks whose targets don't exist.

First, some Homebrew aliases:
```
$HOMEBREW_REPOSITORY/Library/Aliases/actor-framework
$HOMEBREW_REPOSITORY/Library/Aliases/o-caml
$HOMEBREW_REPOSITORY/Library/Aliases/shell-fm
```

I suppose with those that someone who engaged in a rename or a deletion of a formula forgot to handle all its aliases.

But with the next two cases, I dunno what's up:

```
$HOMEBREW_REPOSITORY/Library/ENV/pkgconfig/fuse/fuse.pc
$HOMEBREW_REPOSITORY/Library/ENV/pkgconfig/fuse/osxfuse.pc
```

Running `file` on those gives `broken symbolic link to /usr/local/lib/pkgconfig/fuse.pc`.  That doesn't make sense to me, as I have never brewed fuse/osxfuse.

I haven't made a PR because I want to first make sure these five things really are supposed to be deleted.

**Comment by xu-cheng** [comment]
Thanks for catching this. Borken aliases will be fixed in #43602. The fuse links are supposed to be there as they are only used when fuse is installed.

**Comment by mikemcquaid** [comment]
Thanks for the report @chdiza.
